public/app/features/browse-dashboards/components/BrowseView.tsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useEffect, useState } from 'react';
+
+import { Icon, IconButton, Link } from '@grafana/ui';
+import { getFolderChildren } from 'app/features/search/service/folders';
+import { DashboardViewItem } from 'app/features/search/types';
+
+type NestedData = Record<string, DashboardViewItem[] | undefined>;
+
+interface BrowseDashboardsViewProps {
+  folderUID: string | undefined;
+}
+
+export default function BrowseDashboardsView({ folderUID }: BrowseDashboardsViewProps) {
```

Comment:
minor nit: think we [prefer named exports](https://github.com/grafana/grafana/blob/main/contribute/style-guides/frontend.md#exports)